src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs

```diff
@@ -94,33 +101,47 @@ internal static class HttpContextExtensions
         }
 
         // Internal for testing.
-        internal static string ReplaceMvcParameters(string route, HttpContext? context)
+        internal static string ReplaceMvcParameters(string route, HttpContext context)
         {
-            // Return RouteData or Null, marking the HttpContext as nullable since the output doesn't
-            // shows the nullable output.
-            var routeData = context?.GetRouteData();
+            var routeData = context.GetRouteData();
 
-            if (routeData?.Values["controller"]?.ToString() is { } controller)
+            // GetRouteData can return null on netstandard2
+            if (routeData == null)
+            {
+                return route;
+            }
+
+            var values = routeData.Values;
+
+            if (values["controller"] is string controller)
             {
                 route = Regex.Replace(route, "{controller=[^}]+}", controller);
             }
 
-            if (routeData?.Values["action"]?.ToString() is { } action)
+            if (values["action"] is string action)
             {
                 route = Regex.Replace(route, "{action=[^}]+}", action);
             }
 
-            if (routeData?.Values["area"]?.ToString() is { } area)
+            if (values["area"] is string area)
             {
                 route = Regex.Replace(route, "{area=[^}]+}", area);
             }
 
+            if (values["version"] is string version)
+            {
+                route = Regex.Replace(route, "{version:[^}]+}", version);
```

Comment:
Should it be `=` instead of `:` like the others?  Or was this intentional?

```suggestion
                route = Regex.Replace(route, "{version=[^}]+}", version);
```